interpreter/cling/lib/Interpreter/InterpreterCallbacks.cpp

```diff
@@ -80,19 +80,6 @@ namespace cling {
     }
   };
 
-  /// \brief Wraps an ASTDeserializationListener in an ASTConsumer so that
```

Comment:
I'm surprised we don't need this functionality anymore... IIRC this was pushing deserialized decls (`ASTDeserializationListener`) into the `DeclCollector` (`ASTConsumer`).

Reply:
We didn't get any callbacks from DeserializationListener before this change, and this part of code seems out of date.